src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -17,121 +17,204 @@
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <RoslynProjectType>Vsix</RoslynProjectType>
+    <IsProductComponent>true</IsProductComponent>
+    <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot> 
```

Comment:
IsProductComponent tells the VSSDK to generate a manifest.json file in addition to a VSIX.  All vsixes that we are inserting need this attribute